inc/session.class.php

```diff
@@ -520,7 +520,7 @@ static function loadGroups() {
                    LEFT JOIN `glpi_groups` ON (`glpi_groups_users`.`groups_id` = `glpi_groups`.`id`)
                    WHERE `glpi_groups_users`.`users_id`='" . self::getLoginUserID() . "' " .
                          getEntitiesRestrictRequest(" AND ", "glpi_groups", "entities_id",
-                                                    $_SESSION['glpiactiveentities'], true);
+                                                    $_SESSION['glpiactiveentities'], false);
```

Comment:
with 3520a42fa53318ee0c1afe6d747e81ee7ff06c0c commit, i think we shouldn't do this anymore, no ?

Reply:
Indeed, this was not OK anyways :)